alcs-api/src/common/automapper/user.automapper.profile.ts

```diff
@@ -17,36 +18,50 @@ export class UserProfile extends AutomapperProfile {
 
   override get profile() {
     return (mapper) => {
-      createMap(mapper, CreateOrUpdateUserDto, User);
+      createMap(mapper, UpdateUserDto, User);
       createMap(mapper, UserSettingsDto, UserSettings);
       createMap(mapper, UserSettings, UserSettingsDto);
-      createMap(mapper, UserDto, User);
+      createMap(mapper, CreateUserDto, User);
       createMap(
         mapper,
         User,
         UserDto,
         forMember(
           (ud) => ud.initials,
-          mapFrom(
-            (u) =>
-              u.givenName?.charAt(0).toUpperCase() +
-              u.familyName?.charAt(0).toUpperCase(),
-          ),
+          mapFrom((u) => {
+            if (u.givenName && u.familyName) {
+              return (
+                u.givenName?.charAt(0).toUpperCase() +
+                u.familyName?.charAt(0).toUpperCase()
+              );
+            }
+            return u.name.charAt(0);
+          }),
         ),
         forMember(
           (ud) => ud.mentionLabel,
-          mapFrom(
-            (u) =>
-              u.givenName?.charAt(0).toUpperCase() +
-              u.givenName?.slice(1) +
-              u.familyName?.charAt(0).toUpperCase() +
-              u.familyName?.slice(1),
-          ),
+          mapFrom((u) => {
+            if (u.givenName && u.familyName) {
+              return (
+                u.givenName?.charAt(0).toUpperCase() +
+                u.givenName?.slice(1) +
+                u.familyName?.charAt(0).toUpperCase() +
+                u.familyName?.slice(1)
+              );
+            } else {
+              //TODO: how do mentions work for bceid users?
```

Comment:
I think mentions not relevant for bceid users since only staff members can work on board